src/query/storages/fuse/src/operations/read/runtime_filter_prunner.rs

```diff
@@ -105,6 +105,54 @@ pub(crate) fn update_bitmap_with_bloom_filter(
     let method = DataBlock::choose_hash_method_with_types(&[data_type.clone()], false)?;
     let mut idx = 0;
     match method {
+        HashMethodKind::Serializer(method) => {
+            let key_state = method.build_keys_state(&[(column, data_type)], num_rows)?;
+            match key_state {
+                KeysState::Column(Column::Binary(col)) => col.iter().for_each(|key| {
+                    let hash = key.fast_hash();
```

Comment:
I think we can use a memory buffer for this, if we use a hashes array, we can improve the simd usage to improve the performace. But it doesn't matter.

Reply:
Let's deep into it later